static/app/views/releases/list/index.tsx

```diff
@@ -563,23 +579,27 @@ class ReleasesList extends DeprecatedAsyncView<Props, State> {
 
               {this.shouldShowQuickstart ? null : (
                 <SortAndFilterWrapper>
-                  <StyledSmartSearchBar
-                    searchSource="releases"
-                    query={this.getQuery()}
-                    placeholder={t('Search by version, build, package, or stage')}
-                    hasRecentSearches={false}
-                    supportedTags={{
```

Comment:
Looks like we lost this in the conversion - SmartSearchBar should still send `supportedTags`

Reply:
oops thanks